tests/test_core.py

```diff
@@ -953,97 +945,99 @@ def test_setjmp_noleak(self):
     self.do_run(src, r'''ok.''')
 
   def test_exceptions(self):
-      self.set_setting('EXCEPTION_DEBUG', 1)
-      # needs to flush stdio streams
-      self.set_setting('NO_EXIT_RUNTIME', 0)
+    self.set_setting('EXCEPTION_DEBUG', 1)
+    # needs to flush stdio streams
+    self.set_setting('NO_EXIT_RUNTIME', 0)
 
-      self.set_setting('DISABLE_EXCEPTION_CATCHING', 0)
-      self.maybe_closure()
+    self.set_setting('DISABLE_EXCEPTION_CATCHING', 0)
+    self.maybe_closure()
 
-      src = '''
-        #include <stdio.h>
-        void thrower() {
-          printf("infunc...");
-          throw(99);
+    src = '''
+      #include <stdio.h>
+      void thrower() {
+        printf("infunc...");
+        throw(99);
+        printf("FAIL");
+      }
+      int main() {
+        try {
+          printf("*throw...");
+          throw(1);
           printf("FAIL");
+        } catch(...) {
+          printf("caught!");
         }
-        int main() {
-          try {
-            printf("*throw...");
-            throw(1);
-            printf("FAIL");
-          } catch(...) {
-            printf("caught!");
-          }
-          try {
-            thrower();
-          } catch(...) {
-            printf("done!*\\n");
-          }
-          return 0;
+        try {
+          thrower();
+        } catch(...) {
+          printf("done!*\\n");
         }
-      '''
-      self.do_run(src, '*throw...caught!infunc...done!*')
+        return 0;
+      }
+    '''
+    self.do_run(src, '*throw...caught!infunc...done!*')
 
-      self.set_setting('DISABLE_EXCEPTION_CATCHING', 1)
-      self.do_run(src, 'Exception catching is disabled, this exception cannot be caught. Compile with -s DISABLE_EXCEPTION_CATCHING=0')
+    self.set_setting('DISABLE_EXCEPTION_CATCHING', 1)
+    self.do_run(src, 'Exception catching is disabled, this exception cannot be caught. Compile with -s DISABLE_EXCEPTION_CATCHING=0')
 
-      src = '''
-      #include <iostream>
+  def test_exceptions_custom(self):
+    #self.emcc_args.append('--profiling-funcs')
```

Comment:
remove commented-out code

Also why don't we profile-funcs in testing all the time? Probably make life easier